goa/proj.android/jni/Application.mk

```diff
@@ -1,4 +1,5 @@
 APP_STL := gnustl_static
+APP_DEBUG :=1
 
 APP_CPPFLAGS := -frtti -DCC_ENABLE_CHIPMUNK_INTEGRATION=1 -std=c++11 -fsigned-char
```

Comment:
Remove ogg files that are not required

Reply:
I have removed the audio files that are not required